src/components/nimbleButton/NimbleButton.tsx

```diff
@@ -34,6 +34,7 @@ export interface NimbleButtonProps {
   fontWeight?: fontWeight;
   labelColor?: string;
   height?: string;
+  skipTab?: boolean;
```

Comment:
I think we have to add one more optional prop as tabIndex and change the logic like this

tabIndex={skipTab ? -1 : tabindex??undefined}